prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -360,6 +362,30 @@ class Hl7Serializer(val metadata: Metadata) {
                     terser.set(pathSpec, "") // Not at all sure what to do here.
                 }
             }
+            Element.Type.EMAIL -> {
+                if (value.isNotEmpty()) {
+                    setEmailComponent(terser, value, pathSpec, element)
+                }
+            }
+            Element.Type.DATETIME -> {
+                if (element.name.equals("testing_lab_specimen_received_datetime", ignoreCase = true)) {
+                    val normalDate = try {
```

Comment:
Confused. What is this doing?

Reply:
Some states don't care if the received date time is the same as the collected date time, but others want them offset by a few seconds, so I'm manually shifting the received date time slightly so the file will pass validation. I don't like this hack, and I need to put in a comment about the intent here. I'll remove the magic number too.